SU-SWS/stanford_complex_page
# Removed dependency on stanford_paragraph_types.

# READY FOR REVIEW

# Summary
- Removes references to specific paragraph types in order to keep it generic for all types.

# Needed By (Date)
- 2017.06.19

# Urgency
- Minor

# Steps to Test

1. Check out this branch
2. Ensure the module is enabled and a few paragraph types are available
3. Create a new paragraph type 
4. Run `drush features-import stanford_complex_page -y`
5. Ensure all paragraph types, including the new one you just created, are available on a new complex page node form. 

# Affected Projects or Products
- EARTH

# Associated Issues and/or People
- Nothing directly.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
